test/pkg/debug/collector.go

```diff
@@ -142,10 +142,16 @@ func PrintTestTimes(times map[string][]TimeInterval) map[string][][]string {
 }
 
 // WriteTimestamps will create a temp directory and a .csv file for each suite test
+// If the OUTPUT_DIR environment variable is set, we'll print the csvs to that directory.
 func WriteTimestamps(path string, timestamps *TimeIntervalCollector) error {
-	directory, err := os.MkdirTemp("/tmp", "")
-	if err != nil {
-		return err
+	var directory string
+	var err error
+	directory, ok := os.LookupEnv("OUTPUT_DIR")
```

Comment:
Generally, it's a good idea to ensure that environment variables are set at the top-level in startup and then pass the config down through DI or through contexts.

If we scale-out to add more environment variables, it's probably preferable that we would grab them all in the same place rather than grabbing them at runtime